spec/integration/alerts_spec.rb

```diff
@@ -475,11 +557,38 @@ module Hawkular::Alerts::RSpec
     end
   end
 
-  describe 'Alert/Events', :vcr do
-    VCR.configure do |c|
-      c.default_cassette_options = {
-        match_requests_on: [:method, VCR.request_matchers.uri_without_params(:startTime, :endTime)]
-      }
+  describe 'Alert/Events' do
+    before(:all) do
+      # Setup for testing
+      record('Alert/Events', credentials, 'setup') do
+        @client = Hawkular::Alerts::AlertsClient.new(ALERTS_BASE, credentials, options)
+        json = IO.read('spec/integration/alert-resources/events-test-data.json')
+        hash = JSON.parse(json)
+        hash['events'].each do |event|
+          @client.create_event event['id'], event['category'], event['text'], event['extras']
```

Comment:
I see here is the mapping between "extras" and what create_event uses.
It is just a naming issue but I guess is good unify terminology to not change semantic.
First think I thought reading the test json file was that there were an intermediate class with an "extras" field mapped onto an Event on Hawkular Alerts.